### ISSUE TITLE
Use title case for headings

### ISSUE THREAD

**Issue by 5jt** [Dyalog/documentation#30]
Computing prose is full of proper names, such as Microsoft Windows. They are visually distinguished with initial majuscules. Title case hides roper names, losing information for no benefit and occasionally manufacturing an ambiguity. 

    interfaces to Windows
    interfaces to windows

The practice seems to come from American newspapers:

* [Wall Street Journal](https://wsj.com)
* [New York Times](https://www.nytimes.com)

Newspapers in Canada, Europe, Australia and India use sentence case.

* [Sydney Morning Herald](https://www.smh.com.au)
* [Times of India](https://timesofindia.indiatimes.com)
* [Toronto Globe and Mail](https://www.theglobeandmail.com)
* [The Guardian](https://www.theguardian.com)
* [Politiken](https://politiken.dk)
* [Frankfurter Allgemeine Zeitung](https://faz.net)

We should use sentence case for headings.

**Comment by FionaDyalog** [comment]
I disagree. The usage of sentence case vs title case in documentation is very much a case of personal preference. There is also no US vs UK English distinction as such; the newspapers you have selected have adopted conventions, but this is not a standard for UK/US grammar.

Our documentation set has its own style guide, and all our documents use title case. This extends far beyond the ones within this project. The cost of changing them throughout the entire set is huge and I see no benefit to doing so – people who prefer sentence case will be happier, but at the expense of the people who prefer title case.

(There should also be no ambiguity in the example you show as we should always refer to "Microsoft Windows", not just "Windows" in a title.)

-----

You can argue for either based on web searches, but the Chicago and Oxford style guides both call for title case to be used for headings.

In essence (from [grammarly.com](https://www.grammarly.com/blog/sentences/title-case-sentence-case/), which is not definitive but often useful):
- Title case should be used when following MLA and Chicago style guides. Title case is more commonly seen in formal writing, such as essay titles, published books, titles of magazine articles, and other scholarly works.
- Sentence case should be used when using AP style. It is more common in general or casual forms of writing, such as blog headlines, subject lines of emails, personal essays, and social media captions. Though sentence case is considered more informal, it is typical for newspapers to align with this style.
